pyproject.toml

```
@@ -56,3 +66,16 @@ known-first-party = ["ilthermoml"]
 
 [tool.mypy]
 strict = true
+
+# pytest & plugins
```

@chinise I install _only_ the `mypy` group because what I am interesting in is type-checking of the production code. So, I don't want production dependencies be affected by other dev-dependencies (e.g. from `test`).

Type-checking of tests is an option, which does not have to be "strict", especially in CI.

That is why I propose to include:

```suggestion
[[tool.mypy.overrides]]
module = [
  "pytest",
  "semver",
]
ignore_missing_imports = true

[[tool.mypy.overrides]]
module = [
  "tests.*",
]
disallow_untyped_decorators = false

```